.gitlab-ci.yml

```diff
@@ -34,66 +37,82 @@ protos:
     - cd app && ./generate_protos.sh
   rules:
     - changes:
-      - app/
+      - app/*
```

Comment:
The syntax to use for subtrees are `app/**/*` according to the example in https://docs.gitlab.com/ee/ci/yaml/#onlychangesexceptchanges .

Reply:
Thanks. I'm not sure if that excludes stuff just in `app/` though. This features seems a bit buggy, let's see how it goes.